### PR TITLE
zh: add what's new doc for v1.3

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/TOC.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 - 关于 TiDB Operator
   - [简介](tidb-operator-overview.md)
-  - [What's New in v1.2](whats-new-in-v1.2.md)
+  - [What's New in v1.3](whats-new-in-v1.3.md)
```
Comment (ran-huang) [inline zh/TOC.md:6]:
```suggestion
  - [v1.3 新特性](whats-new-in-v1.3.md)
```

File: zh/TOC.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 - 关于 TiDB Operator
   - [简介](tidb-operator-overview.md)
```
Comment (ran-huang) [inline zh/TOC.md:5]: 顺便把 what's new in v1.2/v1.1 从 master 上删了吧

File: zh/whats-new-in-v1.3.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+title: What's New in TiDB Operator 1.3
+---
+
+# What's New in TiDB Operator 1.3
+
+TiDB Operator 1.3 引入了以下关键特性，从扩展性、易用性、安全性等方面帮助你更轻松地管理 TiDB 集群及其周边工具。
+
+## 兼容性改动
+
+- 跨 Kubernetes 部署 TiDB 集群优化。如果使用 v1.3.0-beta.1 及更早版本的 TiDB Operator 跨 Kubernetes 集群部署 TiDB 集群，直接升级 TiDB Operator 会导致集群滚动更新并导致跨 Kubernetes 特性失效。你需要执行以下操作来升级 TiDB Operator：
```
Comment (ran-huang) [inline zh/whats-new-in-v1.3.md:11]:
```suggestion
- 跨 Kubernetes 部署 TiDB 集群优化。如果使用 v1.3.0-beta.1 及更早版本的 TiDB Operator 跨 Kubernetes 集群部署 TiDB 集群，直接升级 TiDB Operator 会导致集群滚动更新并导致跨 Kubernetes 特性失效。如果从更早版本升级 TiDB Operator 到 v1.3，你需要执行以下操作：
```

File: zh/whats-new-in-v1.3.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+## 滚动更新改动
+
+- TiFlash（>= v5.4.0）默认配置优化。TiDB Operator v1.3.0-beta.1 及之后版本针对 TiFlash v5.4.0 及之后版本的默认配置进行了优化，如果使用 v1.2 版本 TiDB Operator 部署了 v5.4 及更新版本的 TiDB 集群，升级 TiDB Operator 到 v1.3.0-beta.1 及之后版本会导致 TiFlash 组件滚动更新。建议在升级 TiDB 集群到 v5.4.0 或更新版本之前，先升级 TiDB Operator 到 v1.3 及以上版本。
+- 特定配置下 TiKV 可能滚动更新。如果部署了 v5.0 及更新版本的 TiDB 集群，并且设置了 `spec.tikv.seperateRocksDBLog: true` 或者 `spec.tikv.separateRaftLog: true`，升级 TiDB Operator 到 v1.3.0-beta.1 及之后版本会导致 TiKV 组件滚动更新。
+
+## 扩展性
+
+- [跨多个 Kubernetes 集群部署一个 TiDB 集群](deploy-tidb-cluster-across-multiple-kubernetes.md)特性 GA，并支持跨 Kubernetes 集群部署[异构集群](deploy-heterogeneous-tidb-cluster.md)
+- 新增 `failover.recoverByUID` 字段，以支持为 TiKV/TiFlash/DM Worker 仅执行一次性的 Recover 操作
+- 支持为 TidbCluster CR 配置 PodManagementPolicy
```
Comment (ran-huang) [inline zh/whats-new-in-v1.3.md:28]: 解释一下 PodManagementPolicy？

File: zh/whats-new-in-v1.3.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+## 兼容性改动
+
+- 跨 Kubernetes 部署 TiDB 集群优化。如果使用 v1.3.0-beta.1 及更早版本的 TiDB Operator 跨 Kubernetes 集群部署 TiDB 集群，直接升级 TiDB Operator 会导致集群滚动更新并导致跨 Kubernetes 特性失效。你需要执行以下操作来升级 TiDB Operator：
+
+  1. 更新 CRD。
+  2. 修改 TidbCluster 定义将 `spec.acrossK8s` 字段设置为 `true`。
+  3. 升级 TiDB Operator。
+
+- 弃用 Pod `ValidatingWebhook` 和 `MutatingWebhook`。如果使用 v1.2 及更早版本的 TiDB Operator 在集群部署了 Webhook，并启用了 Pod `ValidatingWebhook` 和 `MutatingWebhook`，升级 TiDB Operator 到 v1.3.0-beta.1 及之后版本，Pod `ValidatingWebhook` 和 `MutatingWebhook` 被删除，但这不会对 TiDB 集群管理产生影响，也不会影响正在运行的 TiDB 集群。
```
Comment (csuzhangxc) [inline zh/whats-new-in-v1.3.md:17]: 这种有没有必要在 Operator 的升级文档里也提一下

**Review by csuzhangxc — APPROVED** [review]

File: zh/whats-new-in-v1.3.md (hunk 4 above)
Comment (csuzhangxc) [reply zh/whats-new-in-v1.3.md:28]: 要不 `podManagementPolicy` 链接到 K8s 文档？

File: zh/whats-new-in-v1.3.md (hunk 5 above)
Comment (DanielZhangQD) [reply zh/whats-new-in-v1.3.md:17]: ok

File: zh/TOC.md (hunk 2 above)
Comment (DanielZhangQD) [reply zh/TOC.md:5]: Done.

File: zh/upgrade-tidb-operator.md [hunk 6]
```diff
@@ -8,6 +8,16 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/upgrade-tidb-operator/']
 
 本文介绍如何升级 TiDB Operator 到指定版本。你可以选择[在线升级](#在线升级)或[离线升级](#离线升级)。
 
+> **注意：**
+>
+> - 跨 Kubernetes 部署 TiDB 集群优化。如果使用 v1.3.0-beta.1 及更早版本的 TiDB Operator 跨 Kubernetes 集群部署 TiDB 集群，直接升级 TiDB Operator 会导致集群滚动更新并进入异常状态。如果从更早版本升级 TiDB Operator 到 v1.3，你需要执行以下操作：
```
Comment (ran-huang) [inline zh/upgrade-tidb-operator.md:13]:
```suggestion
> 如果使用 v1.3.0-beta.1 及更早版本的 TiDB Operator 跨 Kubernetes 集群部署 TiDB 集群，直接升级 TiDB Operator 会导致集群滚动更新并进入异常状态。如果从更早版本升级 TiDB Operator 到 v1.3，你需要执行以下操作：
```

**Review by csuzhangxc — APPROVED** [review]

File: zh/upgrade-tidb-operator.md [hunk 7]
```diff
@@ -8,6 +8,16 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/upgrade-tidb-operator/']
 
 本文介绍如何升级 TiDB Operator 到指定版本。你可以选择[在线升级](#在线升级)或[离线升级](#离线升级)。
 
+> **注意：**
```
Comment (ran-huang) [inline zh/upgrade-tidb-operator.md:11]: 写一个单独的小节吧：

## 升级注意事项

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

File: zh/whats-new-in-v1.3.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+## 滚动更新改动
+
+- TiFlash（>= v5.4.0）默认配置优化。TiDB Operator v1.3.0-beta.1 及之后版本针对 TiFlash v5.4.0 及之后版本的默认配置进行了优化，如果使用 v1.2 版本 TiDB Operator 部署了 v5.4 及更新版本的 TiDB 集群，升级 TiDB Operator 到 v1.3.0-beta.1 及之后版本会导致 TiFlash 组件滚动更新。建议在升级 TiDB 集群到 v5.4.0 或更新版本之前，先升级 TiDB Operator 到 v1.3 及以上版本。
+- 特定配置下 TiKV 可能滚动更新。如果部署了 v5.0 及更新版本的 TiDB 集群，并且设置了 `spec.tikv.seperateRocksDBLog: true` 或者 `spec.tikv.separateRaftLog: true`，升级 TiDB Operator 到 v1.3.0-beta.1 及之后版本会导致 TiKV 组件滚动更新。
```
Comment (just1900) [inline zh/whats-new-in-v1.3.md:22]:
```suggestion
- 特定配置下 TiKV 可能滚动更新。如果部署了 v5.0 及更新版本的 TiDB 集群，并且设置了 `spec.tikv.seperateRocksDBLog: true` 或者 `spec.tikv.separateRaftLog: true`，升级 TiDB Operator 到 v1.3.0-beta.1 及之后版本会导致 TiKV 组件滚动更新。
- 为了支持在 TidbMonitor CR 中配置 Prometheus 分片，升级 TiDB Operator 到 v1.3.0-beta.1 及之后版本会导致 TidbMonitor Pod 删除重建。
```

**Review by ran-huang — APPROVED** [review]

**Review by csuzhangxc — APPROVED** [review]

**Pull request merged**